skimage/restoration/_rolling_ball.py

```diff
@@ -53,10 +53,20 @@ def rolling_ball(image, *, radius=100, kernel=None, nansafe=False, num_threads=N
     noise). If this is a problem in your image, you can apply mild
     gaussian smoothing before passing the image to this function.
 
+    This algorithm's complexity is polynomial in the radius, with degree equal
+    to the image dimensionality (a 2D image is N^2, 3D image is N^3 etc.),
+    so it can take a long time as the radius grows beyond 30 or so [2]_ [3]_.
```

Comment:
```suggestion
    so it can take a long time as the radius grows beyond 30 or so ([2]_, [3]_).
```

Reply:
I saw b520395a00e0bc4fc and it would be consistent with
```bash
skimage/segmentation/_watershed.py
145:    This function implements a watershed algorithm [2]_ [3]_ that apportions
```
but there are several instances of `[1]_, [2]_.` and one of `([1]_, [2]_, [3]_).` (hence my suggestion).